modules/global-env.d.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
+/// <reference path="apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts" />
+/// <reference path="apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.d.ts" />
```

Comment:
I don't think we need to add this file here, we only want to reference "global" definitions here. This index.d.ts should get picked up whenever someone imports the source js file